include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -99,10 +99,10 @@ class BrushedMotorInterruptHandler {
         if (old_control_pwm != current_control_pwm) {
             driver_hardware.update_pwm_settings(current_control_pwm);
         }
-        return move_delta;
     }
 
     void execute_active_move() {
+        int32_t move_delta = 0;
```

Comment:
move this into the case at line 120? otherwise it's unused, and it just feels very k&r up here

Reply:
Complier complains that it "Jumps the case statement" if you put it down there. Bothers me too

Reply:
needs braces in the case statement if you're declaring a variable:
```
case MoveStopCondition::encoder_position: {
int32_t move_delta //...
break;
}